src/features/Header/index.tsx

```diff
@@ -1,27 +1,72 @@
-import { useNavigate } from 'react-router-dom';
+import '../Header/index.css';
+import Logo from '../../components/Logo';
 import Button from '../../components/Button';
+import BasicMenu from '../../components/BasicMenu';
+import { useNavigate } from 'react-router-dom';
 
 const Header = (): JSX.Element => {
   const navigate = useNavigate();
+  const onClick = (path: string): void => {
+    navigate(path, {
+      replace: true,
```

Comment:
I think you can remove this. When the user is in login, he can't go back to the main page by pressing the previous page button on the browser. I think it should be possible to go back using the back button

Reply:
I'm talking here about the `replace: true`. I'm not sure that the new page should replace the old one. So I think you should have `replace: false` here.